tests/e2e/kserve_test.go

```diff
@@ -44,14 +47,14 @@ func kserveTestSuite(t *testing.T) {
 	t.Run("Validate component enabled", componentCtx.ValidateComponentEnabled)
 	t.Run("Validate component spec", componentCtx.validateSpec)
 	t.Run("Validate component conditions", componentCtx.validateConditions)
-	t.Run("Validate FeatureTrackers", componentCtx.validateFeatureTrackers)
 	t.Run("Validate model controller", componentCtx.validateModelControllerInstance)
 	t.Run("Validate operands have OwnerReferences", componentCtx.ValidateOperandsOwnerReferences)
-	t.Run("Validate Kserve owns FeatureTrackers", componentCtx.ValidateKserveOwnsFeatureTrackers)
-	t.Run("Validate FeatureTrackers own children", componentCtx.ValidateFeatureTrackersOwnChildren)
-	t.Run("Validate KnativeServing Structure", componentCtx.ValidateKnativeServingStructure)
+	t.Run("Validate no FeatureTracker OwnerReferences", componentCtx.ValidateNoFeatureTrackerOwnerReferences)
+	t.Run("Validate no Kserve FeatureTrackers", componentCtx.ValidateNoKserveFeatureTrackers)
```

Comment:
should we have a test that validates feature tracker are removed ?

Reply:
Are you suggesting a different test case than this "Validate no Kserve FeatureTrackers", or that this one is not necessary and maybe could be removed?

Reply:
maybe I missed it, but I don't see a test that ensure fts are deleted if present

Reply:
You could maybe create some fake FTs before enabling the Kserve and that would be enough I guess, since you are testing already for the absence of those FTs.  But I leave it to you, I won't make this as a strong requirement for the PR to move forward.

Reply:
It turns out that there was one FT being created already! I've made it clearer by creating all 4 FTs before enabling the component. This `ValidateNoKserveFeatureTrackers` is then enough of a test that they are being deleted.